src/client/src/translations/whitelist_en.json

```diff
@@ -77,6 +77,11 @@
   "cosmosResourceModule.saveChanges",
   "cosmosResourceModule.createCosmosRes",
   "cosmosResourceModule.internalName",
+  "cosmosResourceModule.subscriptionSubLabel",
+  "cosmosResourceModule.resourceGroupSubLabel",
+  "cosmosResourceModule.accountNameSubLabel",
+  "cosmosResourceModule.locationSubLabel",
+  "cosmosResourceModule.apiSubLabel",
```

Comment:
I think these are automatically generated. Should we remove them?

Reply:
Removed :)